bndtools.core/bnd.bnd

```diff
@@ -10,8 +10,10 @@ Bundle-Localization: plugin
 
 # 1. JUNit and Felix Shell are optional dependencies.
 # 2. Disable ALL Eclipse split package attributes, to ensure we import from the "aggregator" bundle(s).
-Import-Package: junit.framework;resolution:=optional,\
+Import-Package: \
+	junit.framework;resolution:=optional,\
 	org.apache.felix.shell;resolution:=optional,\
+	com.google.common.collect;version=16.0.0,\
```

Comment:
I see that org.bndtools.templating uses google collections because the mustache templating code uses it, but it uses -conditionalpackage to include the used packages as implementation detail to avoid importing the packages causing a runtime dependencies on another bundle.

Reply:
This is a bit moot now, as I have removed the Guava dependency by changing from the Guava multimap implementation to the Bnd one. However:

1. I would not have pulled in guava were it not for the fact that there was already another bundle in the workspace that was using it.
2. While I can understand dependency fanout being an issue for Bnd, I would argue that it is much less of an issue for Bndtools. The latter is deployed in the context of an Eclipse installation, which already has a large number of dependencies installed (and for this example in particular, Guava is already likely to be among those dependencies, so we wouldn't be adding to the clutter).